LICENSE

```diff
@@ -291,3 +291,7 @@ The text of each license is also included at licenses/LICENSE-[project].txt.
      (MIT License) dagre-d3 (https://github.com/cpettitt/dagre-d3)
      (MIT License) sorttable (https://github.com/stuartlangridge/sorttable)
      (MIT License) boto (https://github.com/boto/boto/blob/develop/LICENSE)
+     (MIT/GPL License) datatables (http://datatables.net/license)
```

Comment:
It doesn't matter much since it's MIT or GPL, and I think this 'notation' was started by the automated output of a Maven plugin.

This is sufficient for MIT-licensed code. I'm a bit concerned about pulling in a lot of third party source just for this feature, but hey.

It looks like you're pulling in an inconsistent set of jquery files though. 1.11.1 vs 1.10.4? or maybe I'm misreading this. They show up as 1-line diffs.